src/runtime/Makefile

```diff
@@ -238,6 +244,7 @@ DEFVALIDENTROPYSOURCES := [\"/dev/urandom\",\"/dev/random\",\"\"]
 
 DEFDISABLEBLOCK := false
 DEFSHAREDFS_CLH_VIRTIOFS := virtio-fs
+DEFSHAREDFS_CLH_SNP_VIRTIOFS := virtio-fs
```

Comment:
Do we need this one? CLH does not support virtiofs with SNP enabled

Reply:
I thought we temporarily won't use virtiofs. If we never use virtio-fs for confidential_guest then it can be removed

Reply:
DEFSHAREDFS_CLH_SNP_VIRTIOFS has been removed